apps/api/src/app/change/usecases/promote-message-template-change/promote-message-template-change.ts

```diff
@@ -14,6 +14,15 @@ export class PromoteMessageTemplateChange {
 
     const newItem = command.item as MessageTemplateEntity;
 
+    const feedDev = await this.feedRepository.findOne({
+      _id: newItem._feedId,
+    });
```

Comment:
@djabarovgeorge should we add _organizationId here aswell?

Reply:
@ainouzgali

Reply:
Yes, will do